src/renderers/webgl/WebGLTextures.js

```diff
@@ -1372,9 +1375,42 @@ function WebGLTextures( _gl, extensions, state, properties, capabilities, utils,
 
 	}
 
+	function updateRoughnessMap( material ) {
+
+		const { roughnessMap, normalMap } = material;
+
+		// ignore PBR materials without roughness and normals maps
+
+		if ( roughnessMap === null || normalMap === null || roughnessMap.generateMipmaps === false ) return;
```

Comment:
Is it possible for the normal and roughness maps to use different UV coordinates? If so, we should disable this then too.

Reply:
They both use the first set of texture coordinates and the same texture transform.